client/src/js/services/ModalService.js

```diff
@@ -279,4 +282,23 @@ function ModalService(Modal) {
       return instance.result;
     }
 
+    /** Find by Date interval modal */
+    function openDateInterval(request) {
+
+      var params = angular.extend(modalParameters, {
+        templateUrl  : 'partials/templates/modals/dateInterval.tmpl.html',
+        controller   : 'DateIntervalModalController',
+        controllerAs : '$ctrl',
+        size         : 'xs',
```

Comment:
Why is this size `xs` when most other modals are `md`?  Does it fit well with the application style?